yt/python/yt/wrapper/py_wrapper.py

```diff
@@ -756,3 +759,184 @@ def with_formats(func, input_format=None, output_format=None):
     if output_format is not None:
         _set_attribute(func, "output_format", output_format)
     return func
+
+
+class DockerRespawner:
+    def __init__(
+            self,
+            image,
+            target_platform,
+            docker,
+            python,
+            env=None,
+            main_scipt_path=None,
+            cwd=None,
+            homedir=None,
+            python_lib_paths=None,
+            mount=None,
+            need_sudo=False,
+    ):
+        self._image = image
+        self._platform = target_platform
+        self._docker = docker
+        self._env = env
+        if self._env is None:
+            # by default, we forward only YT-specific variables
+            self._env = {
+                key: value
+                for key, value in os.environ.items()
+                if key.startswith('YT_')
+            }
+        self._main_script_path = main_scipt_path if main_scipt_path is not None else os.path.abspath(sys.argv[0])
+        self._cwd = cwd if cwd is not None else os.path.abspath(os.getcwd())
+        self._homedir = homedir if homedir is not None else os.path.expanduser("~")
```

Comment:
Let use `get_home_dir` here: https://github.com/ytsaurus/ytsaurus/blob/main/yt/python/yt/wrapper/common.py#L279

Reply:
Discussed in person.

get_home_dir returns None if it can't get homedir.
But here we should work only if we CAN get it. The ability to mount the home directory is our assumption for the normal work of a user's script. Otherwise a user can set mount points in params.